app/api/src/schemas/opportunity_config.py

```diff
@@ -0,0 +1,26 @@
+from random import random
+from src.db import models
+from src.tests.utils.utils import random_lower_string
+
+
+class RequestExample:
+    @property
+    def oportunity_study_area_config(self):
+        return {
+            "sensitivity": None,
+            "multiple_entrance": True,
+            "opportunity_group_id": 14,
```

Comment:
I think this test could fail if the opportunity_group_id does not exist. The ID is currently created as a serial. So it could be that there will be no ID=14. Eventually we should read a random opportunity_group_id and use this one for the test.

Reply:
It runs into circular import when importing crud to get the scenario id. I should find a way to solve it.

Reply:
:white_check_mark: resolved circular import.

Reply:
:heavy_check_mark: fixed